pulp_smash/tests/pulp3/pulpcore/api_v3/test_api_docs.py

```diff
@@ -0,0 +1,39 @@
+# coding=utf-8
+"""Test related to the api docs page."""
+import unittest
+
+from requests.exceptions import HTTPError
+
+from pulp_smash import api, config, selectors, utils
+from pulp_smash.tests.pulp3.constants import API_DOCS_PATH
+from pulp_smash.tests.pulp3.pulpcore.utils import set_up_module as setUpModule  # noqa pylint:disable=unused-import
+
+
+class ApiDocsTestCase(unittest.TestCase, utils.SmokeTest):
+    """Test whether API auto generated docs are available.
+
+    This test targets the following issues:
+
+    * `Pulp #3552 <https://pulp.plan.io/issues/3552>`_
+    * `Pulp Smash #893 <https://github.com/PulpQE/pulp-smash/issues/893>`_
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        """Create class-wide variables."""
+        cls.cfg = config.get_config()
+        if selectors.bug_is_untestable(3552, cls.cfg.pulp_version):
+            raise unittest.SkipTest('https://pulp.plan.io/issues/3552')
+        cls.client = api.Client(cls.cfg, api.safe_handler)
+
+    def test_01_access_api_docs(self):
+        """Test whether the API auto generated docs page is available."""
+        self.client.get(API_DOCS_PATH)
+
+    def test_02_http_method(self):
```

Comment:
These methods don't have any ordering, do they? Drop the numbers from the test method names.